### DISCUSSION TITLE
Set icons for ALL folders and files

### DISCUSSION THREAD

**Discussion by craighagerman** [FlorianWoelki/obsidian-icon-folder#304] [General]
Is there a way to specify an icon to use for all folders and another to use for all files and have that automatically applied to the whole vault. I have a huge vault and can't imagine individually selecting icons for each.

**Comment by FlorianWoelki** [comment]
Does maybe [this documentation](https://florianwoelki.github.io/obsidian-iconize/files-and-folders/custom-rules.html#having-a-default-icon) help?
